examples/boids/src/main.rs

```diff
@@ -1,7 +1,10 @@
 use settings::Settings;
 use simulation::Simulation;
-use slider::Slider;
-use yew::{html, Component, ComponentLink, Html, ShouldRender};
+use slider::Slider as LegacySlider;
+use yew::component::{Component, Context};
+use yew::{html, Html, Legacy, ShouldRender};
+
+type Slider = Legacy<LegacySlider>;
```

Comment:
I didn't migrate this `Slider` intentionally to show how a project could slowly migrate away from the legacy component trait